[docs] Doc about expand entities in investigation graph (#issue/1865) (#65)
OpenCTI-Platform/docs: docs/usage/pivoting.md

![Investigation workspace](assets/investigation-workspace.png)

> You can see next to them a bullet with a number inside. It is a visual indication that shows you how many entities have relationships with this one.

Reviewer: ```suggestion
You can see next to them a bullet with a number inside. It is a visual indication showing you how many entities are linked to this one and not displayed in the graph yet.
```

> On some nodes, you can see next to them a bullet with a number inside. It is a visual indication that shows you how many entities have relationship with this one.

Reviewer 2: have relationshipS

> On some nodes, you can see next to them a bullet with a number inside. It is a visual indication that shows you how many entities have relationship with this one.

Reviewer 2: have relationshipS

> On some nodes, you can see next to them a bullet with a number inside. It is a visual indication that shows you how many entities have relationship with this one.

Reviewer 3: Why some ? Is not for everyone ?

Author: There is no bullet displayed when nothin to expand, but maybe I can just remove "On some nodes" if it is confusing

Author: Removed

Note that this number is an approximation of the number of entities. That's why there is a `~` next to the number.

> No bullet displayed means: nothing to expand from this node.

Reviewer: ```suggestion
No bullet displayed means there is nothing to expand from this node.
```

## Add and expand an entity

You can add any existing entity of the platform to your investigation.

![Investigation bottom right menu](assets/investigation-bottom-right-menu.png)

Once added, you can select the entity, and see its details in the panel that appears on the right of the screen.

In the same menu as above, right next to "Add en entity", you can expand the selected entity. Clicking on the menu icon open a new window where you can choose which type of entities and relationships you want to expand.

> For each type of entity or relationship, the number of elements that will be added into the investigation graph is displayed in parentheses. This time there is no `~` symbol as the number is no more an approximation.

Reviewer: ```suggestion
For each type of entity or relationship, the number of elements that will be added into the investigation graph is displayed in parentheses. This time there is no `~` symbol as the number is exact.
```

> For each type of entity or relationship, it is displayed in parentheses the number of elements that will be added into the investigation graph. This time there is no `~` symbol as the number is no more an approximation at this step.

Reviewer 2: the number of elements that will be added into the investigation graph is displayed in parentheses

> For each type of entity or relationship, it is displayed in parentheses the number of elements that will be added into the investigation graph. This time there is no `~` symbol as the number is no more an approximation at this step.

Reviewer 2: 'this time' and 'this step' are a bit repetitive in the same sentence

> For each type of entity or relationship, it is displayed in parentheses the number of elements that will be added into the investigation graph. This time there is no `~` symbol as the number is no more an approximation at this step.

Author: Changes done in the commit just pushed
